### ISSUE TITLE
Please accept lowercase 'helo' command, et al.

### ISSUE THREAD

**Issue by lechner** [marlam/msmtp-mirror#45]
Hi,

Is it possible that `msmtp` accepts only upper-case SMTP commands?
```
send: 'ehlo lechner-desktop.us-core.com\r\n'
reply: b'500 Expected EHLO or HELO\r\n'
reply: retcode (500); Msg: b'Expected EHLO or HELO'
send: 'helo lechner-desktop.us-core.com\r\n'
Traceback (most recent call last):
  File "/lcl/lechner/keyring/key-expirations/./upcoming-expirations", line 174, in <module>
    send_reminder(recipient_mailbox, recipient_name, status, server_date, keyid, sender_name, sender_email)
  File "/lcl/lechner/keyring/key-expirations/./upcoming-expirations", line 74, in send_reminder
    s.send_message(message)
  File "/usr/lib/python3.9/smtplib.py", line 927, in send_message
    self.ehlo_or_helo_if_needed()
  File "/usr/lib/python3.9/smtplib.py", line 605, in ehlo_or_helo_if_needed
    (code, resp) = self.helo()
  File "/usr/lib/python3.9/smtplib.py", line 434, in helo
    (code, msg) = self.getreply()
  File "/usr/lib/python3.9/smtplib.py", line 398, in getreply
    raise SMTPServerDisconnected("Connection unexpectedly closed")
smtplib.SMTPServerDisconnected: Connection unexpectedly closed
```
I cannot send mail with Python's `smtplib` in [this program here](https://salsa.debian.org/lechner/key-expirations/-/blob/master/upcoming-expirations#L72-74). No workaround seems to exist. [This thread](https://bugs.python.org/issue29860) about the issue says "RFC 821 defines that [the commands] can be any syntax, such as `mAiL FroM`". Thank you!

Kind regards
Felix Lechner

**Comment by marlam** [comment]
This was already fixed last June in commit 7d2222cfd522efc13fde4df448d834bc6ba2b205, and is included already in 1.8.12 (current is 1.8.15).

**Comment by lechner** [comment]
Great, I tested the change and [requested that it is included](https://bugs.debian.org/985468) in Debian's upcoming _bullseye_ release. Thanks for your help!
